plugins/advanced_regex/icon_advanced_regex/actions/replace/action.py

```diff
@@ -16,9 +16,14 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        flags = shared.constructFlags(params)
-        regex = params.get(Input.IN_REGEX)
-        new = params.get(Input.REPLACE_STRING, "")
-        replace_num = params.get(Input.MAX_REPLACE)
-        replaced = re.sub(regex, new, params.get(Input.IN_STRING), count=replace_num, flags=flags)
+        # START INPUT BINDING - DO NOT REMOVE - ANY INPUTS BELOW WILL UPDATE WITH YOUR PLUGIN SPEC AFTER REGENERATION
+        regex = params.get(Input.IN_REGEX, "")
+        input_string = params.get(Input.IN_STRING, "")
+        replace_string = params.get(Input.REPLACE_STRING, "")
+        replace_number = params.get(Input.MAX_REPLACE)
```

Comment:
Would it be best adding .strip()?

Reply:
I would leave it here as it is to be backward compatible (if any customer uses replace with space or something, same as for regex)